cmd/api/src/database/graphschema.go

```diff
@@ -94,6 +95,66 @@ func (s *BloodhoundDB) GetGraphSchemaExtensionById(ctx context.Context, extensio
 	return extension, nil
 }
 
+// GetGraphSchemaExtensions gets all the rows from the extensions table that match the given SQLFilter. It returns a slice of GraphSchemaExtension structs
+// populated with the data, as well as an integer giving the total number of rows returned by the query (excluding any given pagination)
+func (s *BloodhoundDB) GetGraphSchemaExtensions(ctx context.Context, extensionSqlFilter model.SQLFilter, sort model.Sort, skip, limit int) (model.GraphSchemaExtensions, int, error) {
```

Comment:
SQLFilter is a sin we need to put a stop to, we've just been too stretched thin to start fixing it for old stuff. I'd much rather see us start to create DB CRUD that has an explicit structure for what it expects to be able to filter/sort on and force the service layer to provide the translation. I _HATE_ SQL in the API and service layer and would like us to start avoiding it with prejudice.

Not required to merge, but I'd highly suggest we build things right while we're here. Nothing depends on this interface yet, we have carte blanche to use whatever practices provide the best development experience/follow industry practices